### ISSUE TITLE
Difference of the output and target shape with prodived model

### ISSUE THREAD

**Issue by Badar-e-Alam** [xundaoshi/MUSENILM#1]
Hi,

I have tried reproducing the results using a similar configuration but encountered an error at the following line. I suspect this might be due to the default kernel sizes you use, which could potentially be fixed by adjusting the kernel sizes.

**Problematic code:** 
`
loss = criterion(output_status.permute(0,2,1), target_status)
`
**Quick solution:**
` 
 loss = criterion(output_status.permute(0,2,1)[:,:240,:], target_status))
`

This solution works, but I am concerned about its impact during training. By skipping some values, I wonder if it might affect the training process. However, over multiple epochs with randomization, these chunks would probably be compared at some point.
